app/views/camps/_form.haml

```diff
@@ -330,7 +330,15 @@
     = t("form_project_management")
   .h3
     = form.label t("form_project_management_crew")
-  = t("form_project_management_crew_desc")
+  
+  - if Rails.configuration.x.firestarter_settings["crewsize"]
```

Comment:
I'd try to have some sort of convention for using names with and without underscore,
I'm changing this to crew_size for now.